### PR TITLE
Connect routes

### PR DESCRIPTION
**Pull request by GingerBoy14**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -9,6 +9,19 @@
       content="Web site created using create-react-app"
     />
     <title>Senseteq expences</title>
+    <style>
```
Comment (alexpassss) [inline public/index.html:12]: Move to the stylesheets 
Add scss modules to the project

File: src/app/constants/collection.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const collection = {
+  USERS: 'users',
+  WISH: 'wish',
+  CART: 'cart',
+  PRODUCTS: 'products',
+  PURCHASE: 'purchase',
+  LOGS: 'logs',
+  NOTIFICATION: 'notification'
```
Comment (alexpassss) [inline src/app/constants/collection.js:8]: Why not notificationS?

File: src/app/constants/collection.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const collection = {
+  USERS: 'users',
+  WISH: 'wish',
+  CART: 'cart',
```
Comment (alexpassss) [inline src/app/constants/collection.js:4]: Use plural for for collection names

File: src/app/constants/collection.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  PRODUCTS: 'products',
+  PURCHASE: 'purchase',
+  LOGS: 'logs',
+  NOTIFICATION: 'notification'
+}
+
+export { collection }
```
Comment (alexpassss) [inline src/app/constants/collection.js:11]: Use default export

File: src/app/constants/routePaths.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+const ROUTES_PATHS = {
+  LOGIN: '/login',
+  MEMBER_SHOW: '/member/:id',
```
Comment (alexpassss) [inline src/app/constants/routePaths.js:3]: members/:id

File: src/app/constants/routePaths.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  MEMBER_EDIT: '/member/:id/edit',
```
Comment (alexpassss) [inline src/app/constants/routePaths.js:4]: Same here

File: src/app/constants/routePaths.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  NOTIFICATIONS_ALL: '/notifications',
+  CATEGORIES_ALL: '/categories',
+  LOGS_ALL: '/logs',
+  WISHES_ALL: '/wishes',
+  CART_ALL: '/carts',
+  STATISTICS_ALL: '/statistics',
+  SETTINGS: '/settings'
+}
+
+export { ROUTES_PATHS }
```
Comment (alexpassss) [inline src/app/constants/routePaths.js:20]: Default export

**Review by alexpassss — CHANGES_REQUESTED** [review]

**Review by okrdima — APPROVED** [review]

**Pull request merged**